src/components/card/card.js

```diff
@@ -17,6 +20,14 @@ class Card extends mixin(createComponent, initComponentBySearch) {
     this.element.addEventListener('keydown', event => {
       this._cardKeyPress(event);
     });
+    // if (__DEV__) {
```

Comment:
@joshblack Looks that you are experimenting `warning` NPM module. Is it correct? If so - Please ping me if you finished with this - Thanks!

Reply:
Hi @asudoh! Yeah was planning on tackling that today, although it seems like there isn't a "DEV" mode in carbon-components right now to communicate these kinds of warnings. Is that correct? Would hate to include `warning` or `console.error` or something and have it ship to prod.